documentation/upgrading_to_bolt_3.md

```diff
@@ -83,3 +83,62 @@ To migrate your system- or user-level Bolt configuration, make the following cha
     - `apply_settings` to `apply-settings`
     - `plugin_hooks` to `plugin-hooks`
 1. Rename the file to `bolt-project.yaml`.
+
+## Upgrading the puppet-agent package on targets
+
+Starting in Bolt 3.0, Bolt no longer supports puppet-agent versions earlier than
+6.0.0. While applying Puppet code to targets with an earlier version of the
+puppet-agent package might still succeed, Bolt does not guarantee compatibility.
+
+To upgrade the puppet-agent package version installed on a target, you can
+run the `puppet_agent::install` task, which is included in Bolt packages.
+
+- To update to the latest version of the puppet-agent package:
+
+  _\*nix shell command_
+
+  ```shell
+  bolt task run puppet_agent::install --targets <TARGETS> version=latest
+  ```
+
+  _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltTask -Task puppet_agent::install -Targets <TARGETS> version=latest
+  ```
+
+- To update to a specific version of the puppet-agent package:
+
+  ```shell
+  bolt task run puppet_agent::install --targets <TARGETS> version=<VERSION> collection=<COLLECTION>
+  ```
+
+  _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltTask -Task puppet_agent::install -Targets <TARGETS> version=<VERSION> collection=<COLLECTION>
+  ```
+
+  For a list of available collections, see the [task documentation](https://forge.puppet.com/modules/puppetlabs/puppet_agent/tasks).
```

Comment:
Suggest running `bolt task show puppet_agent::install`? That enumerates available collections
```suggestion
  For task documentation, including a list of available collections, run `bolt task show puppet_agent::install` or `Get-BoltTask -Name puppet_agent::install`.
```